hardhat/tasks/deploy.ts

```diff
@@ -0,0 +1,161 @@
+import { task } from 'hardhat/config';
```

Comment:
Is there a reason to use underscores instead of camelCase for functions in this file?